packages/react-app/src/hooks/useManualClaim.js

```diff
@@ -41,21 +41,22 @@ const getMessage = async (homeProvider, homeAmbAddress, txHash) => {
   const messageHash = utils.solidityKeccak256(['bytes'], [msgData]);
 
   const abi = [
-    'event CollectedSignatures(address authorityResponsibleForRelay, bytes32 messageHash, uint256 NumberOfCollectedSignatures)',
+    'function requiredSignatures() public view returns (uint256)',
+    'function numMessagesSigned(bytes32 _message) public view returns (uint256)',
     'function signature(bytes32 _hash, uint256 _index) public view returns (bytes)',
   ];
   const homeAMB = new Contract(homeAmbAddress, abi, homeProvider);
-  let events = await homeAMB.queryFilter('CollectedSignatures', 12123723);
-  events = events.filter(x => x.args.messageHash === messageHash);
-  if (events.length === 0) {
+  const [requiredSignatures, numMessagesSigned] = await Promise.all([
+    homeAMB.requiredSignatures(),
+    homeAMB.numMessagesSigned(messageHash),
+  ]);
+  if (numMessagesSigned < requiredSignatures) {
```

Comment:
Consider to bring these changes inline with the changes from here: https://github.com/maxaleks/omnibridge-eth-bsc/commit/df64df19be3b6e38403faf79f8b2f99e5fb50156